### PR TITLE
Bug 1962261: further adjust memory usage

### PR DESCRIPTION
**Pull request by fpetkovski**
This PR is a follow up of https://github.com/openshift/cluster-monitoring-operator/pull/1158 and further adjusts memory requests for monitoring components.

* [ ] I added CHANGELOG entry for this change.
* [x] No user facing changes, so no entry in CHANGELOG was needed.

### CODE REVIEW

**Comment by simonpasquier** [comment]
I guess you meant "a follow-up of #1158" :)

**Comment by fpetkovski** [comment]
Ah correct :) Updated the description.

**Review by dgrisonnet — COMMENTED** [review]
/lgtm

File: assets/prometheus-operator/deployment.yaml [hunk 1]
```diff
@@ -46,7 +46,7 @@ spec:
         resources:
           requests:
             cpu: 5m
-            memory: 60Mi
+            memory: 150Mi
```
[the remark this answers is not in the document]
Comment (paulfantom) [reply assets/prometheus-operator/deployment.yaml:49]: We added quite a lot of features into prometheus-operator between 4.5 and 4.8, but there wasn't much performance optimization in the meantime.

File: assets/prometheus-operator/deployment.yaml [hunk 2]
```diff
@@ -46,7 +46,7 @@ spec:
         resources:
           requests:
             cpu: 5m
-            memory: 60Mi
+            memory: 125Mi
```
Comment (dgrisonnet) [inline assets/prometheus-operator/deployment.yaml:49]: If I remember correctly, one of the replica of prometheus-operator was often using around 150MiB of memory. That said, wouldn't it be better to readjust the memory request of prometheus-operator to the value of the replica with the highest memory consumption which is 150Mi in this case?

Comment (fpetkovski) [reply assets/prometheus-operator/deployment.yaml]: Good point :+1: Adjusted to 150Mi

**Comment by dgrisonnet** [comment]
/lgtm

@fpetkovski is the hold still meaningful?

**Pull request merged**